Callisto/UserDefaults+LaunchParameters.swift

```diff
@@ -85,3 +88,20 @@ private extension Array {
         return self[index]
     }
 }
+
+extension CommandLine {
+
+    static func parameters(forKey key: String) -> [String] {
```

Comment:
Not relevant for this PR - but if you write new command line tools have a look at `https://github.com/apple/swift-argument-parser`, quite nice utility for getting type safe command line arguments and also documentation…

Reply:
Looks awesome. Will use it in my next project. Thanks for mentioning it